osu.Game.Rulesets.Osu/Mods/OsuModAccuracyChallenge.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Linq;
+using osu.Game.Rulesets.Mods;
+
+namespace osu.Game.Rulesets.Osu.Mods
+{
+    public class OsuModAccuracyChallenge : ModAccuracyChallenge
+    {
+        public override Type[] IncompatibleMods => base.IncompatibleMods.Append(typeof(OsuModAutopilot)).ToArray();
```

Comment:
I don't quite get it.

Reply:
AC needs to reciprocate `OsuModAutopilot`'s incompatibility to pass `TestSceneModValidity`. Is there another way this should be done?

Reply:
Why is autopilot incompatible though?

Reply:
You can't fail with autopilot enabled. It's incompatible with fail condition mods in master.

Reply:
Right, I see.